source/databricks/calculation_engine/tests/calculation/wholesale/test_get_metering_points_and_child_metering_points.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from pyspark.sql import SparkSession
+import pytest
+import tests.calculation.preparation.transformations.metering_point_periods_factory as factory
+from package.calculation.wholesale.get_metering_points_and_child_metering_points import (
+    get_metering_points_and_child_metering_points,
+)
+
+from package.codelists import MeteringPointType
+from package.constants import Colname
+
+
+@pytest.mark.parametrize(
+    "metering_point_type",
+    [
+        MeteringPointType.CONSUMPTION,
+        MeteringPointType.PRODUCTION,
+        MeteringPointType.VE_PRODUCTION,
+        MeteringPointType.NET_PRODUCTION,
+        MeteringPointType.SUPPLY_TO_GRID,
+        MeteringPointType.CONSUMPTION_FROM_GRID,
+        MeteringPointType.WHOLESALE_SERVICES_INFORMATION,
+        MeteringPointType.OWN_PRODUCTION,
+        MeteringPointType.NET_FROM_GRID,
+        MeteringPointType.NET_TO_GRID,
+        MeteringPointType.TOTAL_CONSUMPTION,
+        MeteringPointType.ELECTRICAL_HEATING,
+        MeteringPointType.NET_CONSUMPTION,
+        MeteringPointType.EFFECT_SETTLEMENT,
```

Comment:
Alternatively something similar to this (from the top of my head)
```suggestion
        *[t for t in MeteringPointType if t != MeteringPointType.EXCHANGE]
```
This will be ensure capturing when the set of metering point types change.